### ISSUE TITLE
Email address in the preface gets hyphenated

### ISSUE THREAD

**Issue by AllenDowney** [AllenDowney/ThinkJava#9]
feedback@greenteapress.com gets hyphenated.  Can we avoid that?

**Comment by DanGitOreilly** [comment]
Hi Allen. This should work: 
`<phrase role="keep-together">feedback@greenteapress.com</phrase>`

**Comment by AllenDowney** [comment]
It's currently

```
<emphasis role="bold">feedback@greenteapress.com</emphasis>
```

Can I make it bold _and_ keep it together?

**Comment by AllenDowney** [comment]
This works:

```
<phrase role="keep-together"><emphasis role="bold">feedback@greenteapress.com</emphasis></phrase>
```

**Comment by AllenDowney** [comment]
Resolved with a44a527
